lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Bitdefender.pm

```diff
@@ -0,0 +1,71 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Bitdefender;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('/opt/bitdefender-security-tools/bin/bduitool');
+}
+
+sub doInventory {
+    my (%params) = @_;
+
+    my $inventory = $params{inventory};
+    my $logger    = $params{logger};
+
+    my $antivirus = _getBitdefenderInfo(logger => $logger);
+    if ($antivirus) {
+        $inventory->addEntry(
+            section => 'ANTIVIRUS',
+            entry   => $antivirus
+        );
+
+        $logger->debug2("Added $antivirus->{NAME}" . ($antivirus->{VERSION} ? " v$antivirus->{VERSION}" : ""))
+            if $logger;
+    }
+}
+
+sub _getBitdefenderInfo {
+    my (%params) =  (
+        command => '/opt/bitdefender-security-tools/bin/bduitool get ps',
+        @_
+    );
+
+    my @output = getAllLines(%params)
+        or return;
+
+    my $av = {
+        NAME     => 'Bitdefender Endpoint Security Tools (BEST) for Linux',
+        COMPANY  => 'Bitdefender',
+        ENABLED => 0,
+        UPTODATE => 0,
```

Comment:
In your last commit, you changed UOTODATE default to 0. But uptodate flag should be kept to 1 by default as on l.61 it will be reset to 0 is one of the required field is not set. As now, it will be kept to 0 anyway ;-)
```suggestion
        UPTODATE => 1,
```

Reply:
OK. I thought it was better default to 0.